covid19-dashboard/main.py

```diff
@@ -0,0 +1,147 @@
+    }
+
+    print("Getting perWeek Data")
+    weekly = {
+        'cases': covid.get_cases_for_given_range_alone(region=region, most_recent_date=date, time_delta=7).to_dict(),
+        'deaths': covid.get_deaths_for_given_range_alone(region=region, most_recent_date=date, time_delta=7).to_dict()
+    }
+
+    weeklyPerCapita = {
+        'cases': (covid.get_cases_difference_per_capita(region=region, most_recent_date=date, time_delta=7)).to_dict(),
+        'deaths': (covid.get_deaths_difference_per_capita(region=region, most_recent_date=date, time_delta=7)).to_dict()
+    }
+
+    print("Getting weekIncrease Data")
+    weeklyIncrease = {
+        'cases': (covid.get_cases_increase_pct_for_given_dates(region=region, most_recent_date=date, time_delta=7)).to_dict(),
+        'deaths': (covid.get_deaths_increase_pct_for_given_dates(region=region, most_recent_date=date, time_delta=7)).to_dict()
+    }
+
+    print("Getting perMonth Data")
+    monthly = {
+        'cases': covid.get_cases_for_given_range_alone(region=region, most_recent_date=date, time_delta=30).to_dict(),
+        'deaths': covid.get_deaths_for_given_range_alone(region=region, most_recent_date=date, time_delta=30).to_dict()
+    }
+
+    monthlyPerCapita = {
+        'cases': (covid.get_cases_difference_per_capita(region=region, most_recent_date=date, time_delta=30)).to_dict(),
+        'deaths': (covid.get_deaths_difference_per_capita(region=region, most_recent_date=date, time_delta=30)).to_dict()
+    }
+
+    print("Getting weekIncrease Data")
+    monthlyIncrease = {
+        'cases': (covid.get_cases_increase_pct_for_given_dates(region=region, most_recent_date=date, time_delta=30)).to_dict(),
+        'deaths': (covid.get_deaths_increase_pct_for_given_dates(region=region, most_recent_date=date, time_delta=30)).to_dict()
+    }
+
+    print("Getting absoluteCumulative Data")
+    absoluteCumulative = {
+        'cases': covid.get_cumulative_cases_for_given_date(region=region, date=date).to_dict(),
+        'deaths': covid.get_cumulative_deaths_for_given_date(region=region, date=date).to_dict()
+    }
+
+    print("Getting cumulativePerCapita Data")
+    cumulativePerCapita = {
+        'cases': (covid.get_cumulative_cases_per_capita_for_given_date(region=region, date=date)).to_dict(),
+        'deaths': (covid.get_cumulative_deaths_per_capita_for_given_date(region=region, date=date)).to_dict()
+    }
+
+    return {
+        "daily": daily,
+        "dailyPerCapita": dailyPerCapita,
+        "dailyIncrease": dailyIncrease,
+        "weekly": weekly,
+        "weeklyPerCapita": weeklyPerCapita,
+        "weeklyIncrease": weeklyIncrease,
+        "monthly": monthly,
+        "monthlyPerCapita": monthlyPerCapita,
+        "monthlyIncrease": monthlyIncrease,
+        "absoluteCumulative": absoluteCumulative,
+        "cumulativePerCapita": cumulativePerCapita
+    }
+
+
+#if __name__ == "__main__":
```

Comment:
still need this for local?

Reply:
i need it in local otherwise i cant start the server. I can remve it if youd like

Reply:
that's what i mean, you can remove the comment symbol "#" here

Reply:
Resolved.